package.json

```diff
@@ -6,14 +6,17 @@
   "scripts": {
     "build": "npm run build:spec",
```

Comment:
nit. I think I would probably make build,  build everything v just spe , and have the specific build for running specific stages as you have here. The rationale being if you're new to the repo, you'd git clone, npm run build . This would make it so you could then run test or launch docs right after, as you level up you'll probably run build:spec or whatever specific thing you're looking to run.

Reply:
Good point.  I've updated the build script to do everything..